server/src/main/java/org/candlepin/resource/CrlResource.java

```diff
@@ -26,55 +25,44 @@
 import com.google.inject.Inject;
 
 import io.swagger.annotations.Api;
-import io.swagger.annotations.ApiOperation;
 import io.swagger.annotations.Authorization;
 
 import java.io.File;
 import java.io.FileInputStream;
 import java.io.FileOutputStream;
 import java.io.IOException;
 import java.math.BigInteger;
-import java.security.cert.CRLException;
 import java.util.LinkedList;
 import java.util.List;
+import java.util.Objects;
 
-import javax.ws.rs.DELETE;
-import javax.ws.rs.GET;
 import javax.ws.rs.Path;
-import javax.ws.rs.Produces;
-import javax.ws.rs.QueryParam;
-import javax.ws.rs.core.Context;
-import javax.ws.rs.core.MediaType;
 import javax.ws.rs.core.Response;
 
 /**
  * CrlResource
  */
 @Path("/crl")
 @Api(value = "crl", authorizations = { @Authorization("basic") })
```

Comment:
`@Path` and `@Api` annotations can be removed.

Reply:
Done.